integration-cli/docker_cli_create_test.go

```diff
@@ -305,3 +305,8 @@ func TestCreateLabelFromImage(t *testing.T) {
 
 	logDone("create - labels from image")
 }
+
+func TestCreateHostnameWithNumber(t *testing.T) {
+	dockerCmd(t, "run", "-h", "web.0", "busybox")
```

Comment:
Yeah, just looks weird without it. can be done just with "hostname" command I think.